keymanapp/keyman
# feat(windows): Remove hotkey related feature flags

Step 1. of #11471
Remove the feature flags that allows the use of cached modifier for
hotkeys. Remove the feature flag that uses the Win 32 API
RegisterHotkeys.
The RegisterHotkey support functions for the Win 32 API and the callback function.

# User Testing

## TEST_LANGUAGE_HOTKEYS
For this test only the left modifier keys will trigger the hotkeys
1. Install Keyman
2. Install somekeyboard layouts Tamil and IPA SIL
3. Open Configuration and assign hotkeys to <kbd>Ctrl</kbd>+<kbd>G</kbd> to Tamil and 
<kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>I</kbd>to IPA SIL
5. Open Notepad and type some text 
6. Switch to Tamil using the Hotkey  press <kbd>Ctrl</kbd>+<kbd>G</kbd>
7. Verify that it has switched to Tamil and type some text
8. Switch to IPA SIL using the Hotkey  press <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>I</kbd>
